Plugin~/Src/MeshSyncTest/TestMath.cpp

```diff
@@ -0,0 +1,49 @@
+#include "pch.h"
+#include "Test.h"
+#include "Common.h"
+
+using namespace ms;
+
+template
+<typename T>
+bool compare(T val, T expected) {
+	auto difference = val - expected;
+
+	for (int i = 0; i < difference.vector_length; i++) {
+		if (fabs(difference[i]) > muEpsilon) {
+			return false;
+		}
+	}
+
+	return true;
+}
+
+bool compare(float val, float expected) {
+	auto difference = val - expected;
+
+	return fabs(difference) < muEpsilon;
+}
+
+TestCase(TestRoundToDecimal)
+{
+	// 2 places:
+	// Round down
+	assert(compare(roundToDecimals(mu::tvec3<float>{ 1.121f, 2.342f, 3.453 }), { 1.13f, 2.35f, 3.46f }) && "Error rounding tvec3<float> to 2 decimal places");
```

Comment:
This doesn't look like it's actually rounding down? Is the comment incorrect or am I misunderstanding something?

Reply:
It always rounds up, I tried std::round but it also has floating point inaccuracies and doesn't create floats that are suitable for hashing. ceil works better and is good enough for this use case. I've removed those comments.

Reply:
If we are using ceil, can we please rename the function to indicate that it is using `ceil()` ? `ceilToDecimal` perhaps ?

Another alternative would be implement the correct rounding, meaning that 
1.121  -> 1.12
1.127 -> 1.13

by using typecasting. See [this page](https://java2blog.com/round-to-2-decimal-places-cpp/#Using_the_ceil_function_to_round_to_2_decimal_places_in_C) for example.

I know that in our current use case, we don't necessarily need this correctness, but someone may use it in the future for other use cases, so making it clear if it does `ceil()` or not will make it less misleading.

Reply:
I've renamed it.